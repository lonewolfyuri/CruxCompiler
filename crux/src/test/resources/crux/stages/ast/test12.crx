// test redefinition

var a: int;
var a: bool;

func main() : void {
  ::println();
}
