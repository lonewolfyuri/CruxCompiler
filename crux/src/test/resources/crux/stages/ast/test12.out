DeclareSymbolError(4,0)[a already exists.]
